src/app/login-page/login-page.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss'],
+})
+export class LoginPageComponent {
+  loginForm: FormGroup;
+  constructor(private fb: FormBuilder, private router: Router) {
+    localStorage.clear();
+    this.loginForm = this.fb.group({
+      login: ['', Validators.required],
+      password: ['', Validators.required],
+    });
+  }
+
+  get login() {
+    return this.loginForm.get('login');
+  }
+
+  get password() {
+    return this.loginForm.get('password');
+  }
+
+  onLoanFormSubmit() {
+    if (
+      this.login?.value == 'admin' &&
+      this.password?.value == 'admin' &&
+      this.loginForm.valid
+    ) {
+      localStorage.setItem(
```

Comment:
I'd really recommend using `sessionStorage`. The difference between `localStorage` and `sessionStorage` is `localStorage` persists through browser restarts and tab opening/closing. We don't want that for an 'auth token'.